Client/Frontend/Widgets/SiteTableViewController.swift

```diff
@@ -105,6 +105,11 @@ class SiteTableViewController: UIViewController, UITableViewDelegate, UITableVie
         tableView.tableFooterView = UIView()
     }
 
+    deinit {
```

Comment:
I think adding a comment here pointing back to the bug would help:

```
// The view might outlive this view controller thanks to animations;
// explicitly nil out its references to us to avoid crashes. Bug 1218826.
```